src/map/mob.c

```diff
@@ -1903,7 +1903,7 @@ void mob_item_drop(struct mob_data *md, struct item_drop_list *dlist, struct ite
 	if( sd == NULL ) sd = map->charid2sd(dlist->third_charid);
 
 	if( sd
-		&& (drop_rate <= sd->state.autoloot || pc->isautolooting(sd, ditem->item_data.nameid))
+		&& ((drop_rate <= sd->state.autoloot || pc->isautolooting(sd, ditem->item_data.nameid)) && !map->list[sd->bl.m].flag.noautoloot)
```

Comment:
Does not need this, instead you can edit
pc_isautolooting to return false if player is on noautoloot

Reply:
but i think the `pc_isautolooting` doesnt cover the `sd->state.autoloot`